packages/components/atoms/f-button/src/components/Button.vue

```diff
@@ -4,8 +4,10 @@
         :class="[
             $style['o-btn'],
             $style[`o-btn--${buttonType}`],
-            $style[`o-btn--size${buttonSizeClassname}`],
-            (isFullWidth ? $style['o-btn--fullWidth'] : '')
+            (isIcon ? $style[`o-icon-size${buttonSizeClassname}`] :
+                $style[`o-btn--size${buttonSizeClassname}`]),
+            (isFullWidth ? $style['o-btn--fullWidth'] : ''),
+            (isIcon ? $style['o-icon'] : '')
```

Comment:
I would change this to be a modifier of the base button component. Then you can chain selectors for the size as well (I'll put that comment below to show how I think that would work)

```suggestion
            (isIcon ? $style['o-btn--icon'] : '')
```